nanome/api/plugin_instance.py

```diff
@@ -23,7 +23,20 @@ def __init__(self):
         pass
 
     def __pseudo_init__(self):
-        self.menu = Menu()
+        self.__menu = Menu() #deprecated
+        self.__set_first = False
+
+    @property
+    def menu(self):
```

Comment:
Did we decide on deprecating that? It kinda makes sense, most plugins have only one window
At least a main_menu could make sense?